docs/governance/employment/contractor-credential/governance.md

```diff
@@ -0,0 +1,172 @@
+ - (c) any errors in the Contractor Credential will be corrected; or
+ - (d) the Contractor Credential will meet your expectations and requirements.
+
+***Limitation of Liability***:
+To the maximum extent permitted by applicable law, under no circumstances will the Province be liable to you, to any other individual or to any entity for any direct, indirect, special, incidental, consequential or other loss, claim, injury or damage, whether foreseeable or unforeseeable (including without limitation claims for damages for loss of profits or business opportunities, use or misuse of, or inability to use, the Contractor Credential, interruptions, deletion or corruption of files, loss of programs or information, errors, defects or delays) arising out of or in any way connected with your use of the Contractor Credential and whether based on contract, tort, strict liability or any other legal theory. The previous sentence will apply even if the Province has been specifically advised of the possibility of any such loss, claim, injury or damage.
+
+## 1.2. Terminology and Notation
+
+Please reference [Glossary - General Trust Over IP Terms](https://trustoverip.github.io/toip/glossary).
+
+**Requirements** include any combination of Machine-Testable Requirements and Human-Auditable Requirements. Unless otherwise stated, all Requirements MUST be expressed as defined in [RFC 2119](https://www.rfc-editor.org/rfc/rfc2119).
+
+- Mandates are Requirements that use a MUST, MUST NOT, SHALL, SHALL NOT, or REQUIRED keyword.
+- Recommendations are Requirements that use a SHOULD, SHOULD NOT, or RECOMMENDED keyword.
+- Options are Requirements that use a MAY or OPTIONAL keyword.
+
+**Machine-Testable Requirements** are those with which compliance can be verified using an automated test suite and appropriate scripting or testing software.
+
+**Rules** are Machine-Testable Requirements that are written in a Machine-Readable language and can be processed by a Rules Engine. They are expressed in a structured rules language as specified by the Governance Framework.
+
+**Human-Auditable Requirements** are those with which compliance can only be verified by an audit of people, processes, and procedures.
+
+**Policies** are Human-Auditable Requirements written using standard conformance terminology. The Policies used in the Governance Framework will use the standard terminology detailed in RFC 2119 keywords. Note that all RFC 2119 keywords have weight from an auditing perspective. An implementer MUST explain why a SHOULD or RECOMMENDED requirement was not implemented and SHOULD explain why a MAY requirement was implemented.
+
+**Specifications** are documents containing any combination of Machine-Testable Requirements and Human-Auditable Requirements needed to produce technical interoperability.
+
+## 1.3. Localization
+
+The standard language for this Governance Framework (GF) is English.
```

Comment:
Since you're using the term Governance Framework much earlier (line 10), I would introduce and apply the acronym earlier, and consistently apply it from there, unless there are cases where you want/need to spell out the whole term for valid reasons. And also being consistent with capitalization/no capitalization.

Reply:
this applies to the CSB doc as well